orangecontrib/text/import_documents.py

```diff
@@ -513,13 +513,17 @@ def _add_metadata(self, corpus: Corpus) -> Corpus:
             or self._meta_data is None
             or (
                 self.META_DATA_FILE_KEY not in self._meta_data.columns
-                and self.CONLLU_META_DATA not in self._meta_data.columns
+                and not any(i in self._meta_data.columns for i in
+                         self.CONLLU_META_DATA)
             )
         ):
             return corpus
 
         if self.is_conllu:
-            df = self._meta_data.set_index(self.CONLLU_META_DATA)
+            # find the first matching column
+            match_id = next((idx for idx in self.CONLLU_META_DATA if idx in self._meta_data.columns),
+                            None)
```

Comment:
Why does this next() have a default value? Isn't the upper condition supposed to filter out those cases? If so, a default value here would only hide potential issues.

Reply:
Fixed.